examples/custom-resources/access-control/README.md

```diff
@@ -1,96 +1,3 @@
-# Access Control
+# Deploy an Access Control policy
```

Comment:
Access Control is a feature and policy is one of the steps so the original title is apt

Reply:
The original title does not describe the intent of the document, and the new one is matched to the website document, which is written as an instruction.